proteus/TwoPhaseFlow/TwoPhaseFlowProblem.py

```diff
@@ -79,9 +87,10 @@ def assert_initialConditions(self,ns_model,nd,initialConditions):
         assert 'vel_v' in initialConditions, 'Provide vel_v in ICs'
         if nd==3:
             assert 'vel_w' in initialConditions, 'Provide vel_w in ICs'
-        assert 'clsvof' in initialConditions, 'Provide clsvof in ICs'
-        if ns_model==1: #rans3p
-            assert 'pressure_increment' in initialConditions, 'Provide pressure_increment in ICs'
+        if ns_model == 1:
+            assert 'clsvof' in initialConditions, 'Provide clsvof in ICs'
+            if ns_model==1: #rans3p
```

Comment:
I think there is an indentation error here.

Reply:
You are right, it should not be there as it can be used with any `ns_model`. I think modifying the `initialConditions` assertions here would be good, because we could use `'clsvof`', '`ncls`' or even no model for the free surface. We could check that all the models that are included in the Problem class have their respective `initialConditions`